### PR TITLE
[4.0] Quickicons

### PR DESCRIPTION
**Pull request by brianteeman**
Change the order of the background clip in the css for the quickicons

Correct PR for #28090

### CODE REVIEW

File: administrator/templates/atum/scss/blocks/_quickicons.scss [hunk 1]
```diff
@@ -19,8 +19,8 @@
       margin: 0 auto;
       font-size: $quickicon-icon-size;
       background: linear-gradient(var(--atum-link-color), #5693e1);
-      -webkit-background-clip: text;
       background-clip: border-box;
+      -webkit-background-clip: text;
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/blocks/_quickicons.scss]: Remove vendor prefix

File: administrator/templates/atum/scss/blocks/_quickicons.scss [hunk 2]
```diff
@@ -37,8 +37,8 @@
       .fas,
       .fab {
         background: linear-gradient(var(--atum-bg-light), $white);
-        -webkit-background-clip: text;
         background-clip: border-box;
+        -webkit-background-clip: text;
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/blocks/_quickicons.scss]: Same as above

File: administrator/templates/atum/scss/blocks/_quickicons.scss [hunk 3]
```diff
@@ -213,8 +213,8 @@
       .fab {
         color: var(--success);
         background: linear-gradient(var(--success), #159c1e);
-        -webkit-background-clip: text;
         background-clip: border-box;
+        -webkit-background-clip: text;
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/blocks/_quickicons.scss]: Same as above

File: administrator/templates/atum/scss/blocks/_quickicons.scss [hunk 4]
```diff
@@ -227,8 +227,8 @@
           &::before {
             color: var(--white);
             background: linear-gradient($white, lighten($green, 40%));
-            -webkit-background-clip: text;
             background-clip: border-box;
+            -webkit-background-clip: text;
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/blocks/_quickicons.scss]: Same as above

**Comment by infograf768** [comment]
Before and after  @C-Lodder proposed corrcetions, it works here Firefox Macintosh

**Comment by infograf768** [comment]
@brianteeman please correct so we can merge asap

**Review by C-Lodder — APPROVED** [review]

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**